Rocket.ChatTests/AuthSettingsModelMappingSpec.swift

```diff
@@ -0,0 +1,43 @@
+//
```

Comment:
Move it to the folder `Rocket.ChatTests/Models/` and rename it to `AuthSettingsSpec`, just to follow the way we're doing with our models tests.